var/spack/repos/builtin/packages/itstool/package.py

```diff
@@ -14,9 +14,13 @@ class Itstool(AutotoolsPackage):
     homepage = "http://itstool.org/"
     url      = "http://files.itstool.org/itstool/itstool-2.0.2.tar.bz2"
 
+    maintainers = ['agoodLANL']
+
+    version('2.0.7', sha256='6b9a7cd29a12bb95598f5750e8763cee78836a1a207f85b74d8b3275b27e87ca')  
     version('2.0.2', sha256='bf909fb59b11a646681a8534d5700fec99be83bb2c57badf8c1844512227033a')
     version('2.0.1', sha256='ec6b1b32403cbe338b6ac63c61ab1ecd361f539a6e41ef50eae56a4f577234d1')
     version('2.0.0', sha256='14708111b11b4a70e240e3b404d7a58941e61dbb5caf7e18833294d654c09169')
     version('1.2.0', sha256='46fed63fb89c72dbfc03097b4477084ff05ad6f171212d8f1f1546ea543978aa')
 
     depends_on('libxml2+python', type=('build', 'run'))
+    conflicts('^python3', when='@:2.0.2')
```

Comment:
There is no `python3` package and better to write as a requirement:
```suggestion
    depends_on('python@2.0:2', when='@:2.0.2')
```